docs/general/faq.md

```diff
@@ -295,16 +295,16 @@ implementers.
 ### Are parachains ephemeral? What happens when a parachain loses the next auction?
 
 Parachains are not ephemeral. As long as someone is keeping the data for a parachain, the parachain
-can move between being a parachain, a parathread, or a separate sovereign chain at different points
-of its lifetime. Especially with parathreads, parachains can be decommissioned to only produce
-blocks when their usage and throughput makes it necessary.
+can move between being a permanent parachain, an
```

Comment:
"Permanent" may be a little misleading, perhaps better to allude to bulk coretime instead (cause either way it has to be renewed).

Reply:
just parachains